### ISSUE TITLE
Minor formatting issues in UG

### ISSUE THREAD

**Issue by ReubenTeng** [theyellowfellow/ped#4]
![image.png](https://raw.githubusercontent.com/theyellowfellow/ped/main/files/c66c0c84-18b1-48af-a805-fa8a8c6eaf99.png)
Extra square bracket after dt/DEADLINE_TIME, but it really is a very very minor error.

**Comment by claracheong4** [comment]
I don't think that this is a typo, the intended effect would be for prefix dd/ there could be a DEADLINE_DATE an optional DEADLINE_TIME with prefix dt/. In other words, dt/DEADLINE_TIME is part of dd/DEADLINE_DATE.
